themes/socialblue/templates/node/node--hero--sky.html.twig

```diff
@@ -169,45 +169,47 @@
 
           {% block metaengage %}
             <div class="meta-engage">
-              <div class="hero-footer__list-left">
-                {% if comment_field_status is not empty %}
-                  <a href="#section-comments" class="badge badge--large"
-                     title="{% trans %}Total amount of comments{% endtrans %}">
+              {% if (comment_field_status is not empty) and (views_count is not empty and views_label) and (content.like_and_dislike) %}
```

Comment:
@agami4 With this change you remove comment and likes from all other existing heroes. So it needs to be fixed in other way - maybe you need to make changes only in challenge hero template.

![image](https://user-images.githubusercontent.com/2241917/102209379-56cc7f80-3ed9-11eb-8c15-addb4a8609e6.png)

Reply:
@ribel, I've checked some nodes and this condition is no needed. I removed condition

Reply:
@ribel, I've created an issue on Drupal.org